kbase-extension/scss/partials/_job-status.scss

```diff
@@ -0,0 +1,130 @@
+.kb-job-status {
+  box-shadow: none;
+
+  &__row {
+    min-height: 40px;
+
+    .show_log {
+      display: none;
+    }
+
+    .selected_log {
+      display: none;
+    }
+
+    &:hover {
+      background: #F2EFEB;
+
+      .show_log {
+        display: inline;
+      }
+    }
+
+    &_selected {
+      background: #CCE5F3;
+
+      &:hover {
+        background: #CCE5F3;
+
+        .show_log {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__icon {
+    margin: 4px;
+
+    &--completed{
+      color: rgb(17, 133, 119);
+    }
+
+    &--error,
+    &--does_not_exist,
+    &--terminated  {
+      color: rgb(210,35,42);
+    }
+
+    &--queued,
+    &--created,
+    &--estimating,
+    &--running {
+      color: rgb(161,185,207);
+    }
+  }
+// panel-heading border none
+  &__table {
+    font-family: Oxygen;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 40px;
```

Comment:
You should be able to use the `font` shorthand to specify these all at once. We also have variables set up in the `modules/_variables.scss` file for typefaces, so this can turn into:

`font: normal 400 $fontsize/40px $typeface-page-text;`

(please substitute in whatever the correct font size is)